### ISSUE TITLE
[serialization] move bitset serialization from frame_object_diff (Trac #1945)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1937]
The frame_object_diff project has a serialization specialization for std::bitset.  Now that we have control of the serialization code, move this to the serialization project:

http://code.icecube.wisc.edu/projects/icecube/browser/IceCube/projects/frame_object_diff/trunk/private/bitset.hpp

Migrated from https://code.icecube.wisc.edu/ticket/1945
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:14:44", 
    "description": "The frame_object_diff project has a serialization specialization for std::bitset.  Now that we have control of the serialization code, move this to the serialization project:\n\nhttp://code.icecube.wisc.edu/projects/icecube/browser/IceCube/projects/frame_object_diff/trunk/private/bitset.hpp", 
    "reporter": "david.schultz", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067284370534", 
    "component": "combo core", 
    "summary": "[serialization] move bitset serialization from frame_object_diff", 
    "priority": "normal", 
    "keywords": "", 
    "time": "2017-01-31T17:04:25", 
    "milestone": "", 
    "owner": "cweaver", 
    "type": "task"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20170131T17:10:21: cweaver commented:

> The serialization library already supports `std::bitset`, although it looks like the implementations may not be compatible. Is the intent to replace the existing implementation with the one in frame_object_diff, and if so, are we sure that nothing is using the existing version?

**Comment by icecube-trac** [comment]
Trac update at 20170131T17:18:41: david.schultz commented:

> Hmm, I didn't know about that. But yes, they aren't compatible at all.  The one in the serialization library stores the bitset at a string of "0101101", which is really inefficient.
> 
> Looking through combo, `std::bitset` is used in two other places, but not for serialization.

**Comment by icecube-trac** [comment]
Trac update at 20170131T17:20:15: cweaver commented:

> I also don't see anywhere else it was being used, but I wanted to double check. I agree that the existing implementation is terrible, so I will be quite happy to replace it.

**Comment by icecube-trac** [comment]
Trac update at 20170131T20:50:43: 
* cweaver changed resolution from "" to "fixed"
* cweaver changed status from "new" to "closed"
* cweaver commented:

> In changeset 153058:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="153058"
> Delete the implementation of bitset serialization here and use the one in the serialization library. 
> 
> Closes #1937
> ```

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:14:44: 
* olivas commented:

> Milestone July 2017 deleted


* olivas changed milestone from "July 2017" to ""
